whois-commons/src/main/java/net/ripe/db/whois/common/etree/NestedIntervalMap.java

```diff
@@ -226,6 +232,17 @@ private List<InternalNode<K, V>> internalFindFirstMoreSpecific(K range) {
         return result;
     }
 
+    private List<InternalNode<K, V>> internalFindMostSpecific(K range){
+        final List<InternalNode<K, V>> result = internalFindAllMoreSpecific(range);
+        if (result.isEmpty()){
+            return Lists.newArrayList();
```

Comment:
No need for this check if you are using stream

Reply:
True